contracts/implementation/PoolCommitter.sol

```diff
@@ -363,17 +363,18 @@ contract PoolCommitter is IPoolCommitter, Ownable {
         IERC20 _token = IERC20(_quoteToken);
         bool approvalSuccess = _token.approve(leveragedPool, _token.totalSupply());
         require(approvalSuccess, "ERC20 approval failed");
-        _token.approve(leveragedPool, _token.totalSupply());
```

Comment:
Why remove this?

Reply:
Look at line 364, there's already a token approval that happens